.github/workflows/release.yaml

```diff
@@ -0,0 +1,85 @@
+name: release
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_REGISTRY: ghcr.io
+  IMAGE_REPO: ${{ github.repository_owner }}/golden-container
+  IMAGE_TAGS: latest
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Needed for signing the container image with GitHub OIDC Token
+      id-token: write
+      # Needed to push to ghcr.io registry
+      packages: write
+
+    outputs:
+      image:  ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_REPO }}
+      digest: ${{ steps.push-image.outputs.digest }}
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Cosign
+      uses: sigstore/cosign-installer@main
+
+    - name: Install Syft
+      uses: anchore/sbom-action/download-syft@v0.14.3
+
+    - name: Log in to ghcr
```

Comment:
@zregvart Do we need this with redhat-actions/push-to-registry below? I guess for cosign? Could leave a comment and nest this next to cosign below...